tests/TestAbyssFixes.py

```
@@ -0,0 +1,807 @@
+        )
+        char.continues_normal_attack = lambda duration: setattr(
+            char, "_fake_time", char._fake_time + duration
+        )
+        char.continues_right_click = lambda *a, **k: None
+        char.is_current_char = True
+        char.is_dead = False
+        char.task.click.side_effect = lambda *a, **k: None
+        return char
+
+    def test_r_cooldown_shared_between_normal_and_burst(self):
+        """普通状态 R 触发后，爆发内 R 冷却仍按全局计时，不会从零开始。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)